client/src/i18n/en-US.json

```diff
@@ -416,7 +416,7 @@
             "true": "Anyone can find this publication.",
             "false": "Publication will be hidden from search."
           },
-          "hint": "Set to publicly visible to allow your publication to be discoverable."
+          "hint": "Set to publicly visible to allow your publication to be discoverable by all users."
```

Comment:
For now, it is true that only registered users of Pilcrow will be able to discover a publication. However, I understand that there is an intention to make the home pages of publications publicly-visible to those who are not registered users of Pilcrow, and the control that this `hint` is for would determine that visibility. 

If you feel this edit should remain for now, then I want to note that this will need to be reverted once the public-visibility of publication home pages is implemented.

Reply:
We can keep this edit for now and just update it later when necessary.